docs/release-notes/8.3.asciidoc

```diff
@@ -123,6 +149,10 @@ NOTE: This is an alternative option for users who have only enabled custom rules
 
 On the next scheduled rule execution, the *Last Response* value for the rule will change to `Pending`, and then to `Active` or `OK`.
 
+*The `matches` operator is not supported for rule exceptions*
+
+* The `matches` operator in the *Add Rule Exception* flyout does not work because wildcard matches are not supported for rule exceptions. Using the `matches` operator will cause rule exceptions and their associated rules to fail. You can restore failed rules by deleting the unsupported exception and refreshing the rule ({issue}136340[#136340]).
```

Comment:
```suggestion
* The `matches` operator in the *Add Rule Exception* flyout does not work because wildcard matches are not supported for rule exceptions. Using the `matches` operator will cause rule exceptions and their associated rules to fail. You can restore failed rules by deleting the unsupported exceptions and refreshing the rules ({issue}136340[#136340]).
```